src/DIRAC/Core/scripts/dirac_generate_crls.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+########################################################################
+# File :   dirac-generate-cas
+# Author : Andrii
+########################################################################
+"""
+Generate a single CA file with all the PEMs
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+import sys
+
+from DIRAC import gLogger
+from DIRAC.Core.Utilities.DIRACScript import DIRACScript as Script
+from DIRAC.Core.Security import Utilities
+
+
+@Script()
+def main():
+    Script.parseCommandLine(ignoreErrors=True)
```

Comment:
why `ignoreErrors=True` ?

Reply:
If I'm not mistaken, this operation is so simple that it does not depend on the state of the configuration. Therefore, I can use the `ignoreErrors` option here and ignore errors when loading the configuration.